tests/pytorch/test_nn.py

```diff
@@ -1483,3 +1483,42 @@ def test_pna_conv(in_size, out_size, aggregators, scalers, delta,
     model = nn.PNAConv(in_size, out_size, aggregators, scalers, delta, dropout,
         num_towers, edge_feat_size, residual).to(dev)
     model(g, h, edge_feat=e)
+
+@pytest.mark.parametrize('k', [3, 5])
+@pytest.mark.parametrize('alpha', [0., 0.5, 1.])
+@pytest.mark.parametrize('norm_type', ['sym', 'row'])
+@pytest.mark.parametrize('multi_label', [True, False])
+@pytest.mark.parametrize('reset', [True, False])
+def test_label_prop(k, alpha, norm_type, multi_label, reset):
+    dev = F.ctx()
+    num_nodes = 5
+    num_edges = 20
+    g = dgl.rand_graph(num_nodes, num_edges).to(dev)
+    labels = th.tensor([0, 2, 1, 3, 0]).long().to(dev)
+    mask = th.tensor([0, 1, 1, 1, 0]).bool().to(dev)
+    model = nn.LabelPropagation(k, alpha, norm_type, multi_label, reset)
+    model(g, labels, mask)
```

Comment:
Need test for two cases:
1) multiclass classification + one-hot labels
2) multitask classification